ospfd/ospf_ri.c

```diff
@@ -1467,6 +1538,8 @@ static uint16_t show_vty_sr_msd(struct vty *vty, struct tlv_header *tlvh)
 {
 	struct ri_sr_tlv_node_msd *msd = (struct ri_sr_tlv_node_msd *)tlvh;
 
+	check_tlv_size(RI_SR_TLV_NODE_MSD_SIZE, "SR Label Range");
```

Comment:
I noticed one more nit - the TLV name is not correct here.

Reply:
Done + rebase on top of master